Plugin/Controller/Cart/Add.php

```diff
@@ -93,7 +93,9 @@ public function __construct(
     public function aroundExecute(\Magento\Checkout\Controller\Cart\Add $subject, callable $proceed)
     {
         $usageId = $this->request->getParam('usage_id');
-        if ($this->request->getParam('usage_category') != $this->getCustomLicenseId()) {
+        $usageCategory = $this->request->getParam('usage_category');
+        if ($usageCategory != $this->getCustomLicenseId() &&
```

Comment:
Seems like this should be an OR condition. If the usage category is not the custom category OR category is not set continue. But maybe I'm not understanding something.
```suggestion
        if ($usageCategory != $this->getCustomLicenseId() ||
```